dts/bindings/spi/st,stm32h7-spi.yaml

```diff
@@ -27,3 +27,7 @@ properties:
     description: |
       (Master SS Idleness) minimum clock inserted between
       start and first data transaction.
+
+  fifo-enabled:
```

Comment:
Maybe it is better to rename it to `fifo-enable` to stay consistent with STM32 serial driver: https://github.com/zephyrproject-rtos/zephyr/blob/346997f8be66565ed8d5fe5d4ff1f8078ff8dec1/dts/bindings/serial/st%2Cstm32-uart-base.yaml#L90

Reply:
I would rather suggest to modify `fifo-enable` to `fifo-enabled`, which is semantically correct in my opinion. I have inspected it and there aren't many occurrences of the "current" `fifo_enable`, so it would be simple to modify. I would understand modifying this PR if `fifo-enable` was used in many places, but it's not the case.

Reply:
`fifo-enable` is actually already used out of tree and in other bindings. I don't think the arguable semantic difference is worth the trouble.